Klipper_Files/Extra module/ercf.py

```diff
@@ -787,6 +795,7 @@ def _unload_from_end_of_bowden(self, length):
         self.toolhead.dwell(0.2)        
         self.toolhead.wait_moves()
```

Comment:
This looks doubled up.

Reply:
There is a huge difference:

- dwell inserts a wait into the toolhead queue.
- Wait_moves waits for the queue to be empty.

Emmo

Reply:
There is already a wait there.

<img width="349" alt="image" src="https://user-images.githubusercontent.com/473577/200203564-8852845a-d9cd-41ad-88cb-f3c4b07f0c70.png">